Foundations/Components/ProfileCircle.swift

```diff
@@ -10,33 +10,45 @@ import SwiftUI
 struct ProfileCircle: View {
     @State private var imagePath: String;
     @State private var profileName: String;
+    @State private var radius: CGFloat;
     
-    init(imagePath: String, profileName: String) {
+    init(_ imagePath: String,
+         profileName: String = "",
+         radius: CGFloat = 100
+    ) {
+        self.radius = radius;
         self.imagePath = imagePath;
         self.profileName = profileName;
     }
     
     var body: some View {
         VStack {
-            ZStack {
-                Image(imagePath)
-                Circle()
+            Image(imagePath)
+                .resizable()
+                .shadow(radius: 10)
+                .overlay(Circle()
                     .inset(by: 0)
-                    .stroke(Color("light-yellow"),
-                            lineWidth: 8)
+                    .stroke(
+                        Color("light-yellow"),
+                        lineWidth: radius / 10
+                    )
+            ).frame(width: radius, height: radius)
+             .padding(.bottom)
+            
+            if (profileName != "") {
+                Text(profileName)
+                    .font(.custom("Inter-Regular",
+                                  size: 13))
+                    .foregroundColor(.black)
+                    .fontWeight(.light)
             }
-                .frame(width: 100, height: 100)
```

Comment:
I was wondering if we could make the circle distance to the text customizable by the user. this would be proven very useful in the screen I am curently developing (feedback cards)

Reply:
Yes, you can just pass the value as a parameter.